modfile/go.mod.tmpl

```diff
@@ -0,0 +1,71 @@
+module {{ .Module }}
+
+go {{ .Go }}
+
+{{- if .Toolchain.Version }}
+
+{{ if .Toolchain.Comment }}// {{ .Toolchain.Comment }}{{ end }}
+toolchain {{ .Toolchain.Version }}
+{{- end }}
+
+{{- if .Replace.Replacements }}
+{{ if .Replace.Comment }}
+// {{ .Replace.Comment }}
+{{- end }}
+replace (
+{{- range .Replace.Replacements }}
+{{ printf "\t" }}{{ . }}
+{{- end }}
+)
+{{- end }}
```

Comment:
I made sure that all of these blocks were as identical as possible, so it should be easy to add additional directives (e.g. `retract`) in the future.